backend/examples/user_custom_properties/test/locale/fr.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Test": "un Test",
+  "just test data": "juste des données de tests",
+  "some parameters": "quelques paramètres",
+  "sub fields": "sous-champs",
+  "First": "1er",
+  "Second": "",
```

Comment:
```suggestion
  "Second": "Deuxième",
```

Reply:
This one will not been fixed as this is just a test example of translation. The fact it is not provided is "a feature".